hyx/retry/manager.py

```diff
@@ -7,24 +7,37 @@
 from hyx.retry.exceptions import AttemptsExceeded
 from hyx.retry.listeners import RetryListener
 from hyx.retry.typing import AttemptsT, BackoffsT
+from hyx.ratelimit.managers import TokenBucketLimiter
 
+class GlobalRetryLimiter:
```

Comment:
`GlobalRetryLimiter` doesn't seem to do much. Do you think its worth introducing it vs. using `TokenBucketLimiter` (or maybe some abstract class/protocol derived from its signature)?

Reply:
That's true. GlobalRetryLimiter *could* be extended to take in different limiting algorithms, unless TokenBucketLimiter algorithm is sufficient for retries?

Reply:
Yeah, seems like the signature of the current limiters are good enough for what we demand right now on the retry side. So I would start small by leveraging the existing stuff and then see if we have any case for introducing more sophisticated interfaces

Reply:
will do